test/OpExtInst.ll

```diff
@@ -0,0 +1,126 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.bc
+; RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
```

Comment:
It would be helpful to have a comment describing the purpose of the test, e.g. "Test that debug info instructions are not inserted inbetween branch and loop merge instructions".

Reply:
Done in a new test